astromon/db.py

```diff
@@ -85,12 +77,10 @@
     ('sim_z', np.float32),
     ('date_obs', 'S20'),
     ('tstart', np.float32),
-    ('fids', 'S20'),
```

Comment:
I assume this was in the old db to try to correlate offsets with fid sets?  It does seem like we've got more facility with quickly getting star catalogs again if that is needed.

Reply:
are you implying that it is ok to remove it?

Reply:
The comment was me mostly trying to guess why it was there before to decide if it is OK to remove it.   But overall seems reasonable to me to remove it.